src/app/transfer-funds/transfer-funds.component.html

```diff
@@ -23,8 +23,8 @@ <h3 class="b-rh-1 b-bold">Transfer Donation Funds</h3>
                         Must be between {{ minimumCreditAmount | exactCurrency:currency }} and {{ maximumCreditAmount | exactCurrency:currency }} inclusive.
                     </p>
 
-                    <div *ngIf="(donor.pending_tip_balance?.gbp || 0) > 0">
-                        <p>Thanks for tipping Big Give {{ ((donor.pending_tip_balance?.gbp || 0) / 100) | exactCurrency:"GBP" }} </p>
+                    <div *ngIf="pendingTipBalance > 0">
```

Comment:
this expression `pendingTipBalance > 0` is used three times - probably worth making it into a function like `donorHasPendingTip: boolean` or something?

Reply:
Done in https://github.com/thebiggive/donate-frontend/pull/1367/commits/5e0cb12d96f390e4dbd7cccaf576d07a84fd7b6a